### PR TITLE
RRT Util Parameter Port

### PR DESCRIPTION
**Pull request by jimtjames**
## Description
Converting the rrt util params to ROS2.

I had issues getting the parameter assignment in the `create_configuration` method to compile, so I ended up moving the parameter assignment and declaration outside the configuration. Let me know what to fix!

### CODE REVIEW

**Review by kasohrab — COMMENTED** [review]
Compiles fine, but fails test-all.

File: soccer/src/soccer/planning/primitives/rrt_util.cpp [hunk 1]
```diff
@@ -15,40 +15,29 @@
 namespace Planning {
 REGISTER_CONFIGURABLE(RRTConfig)
```
Comment (kylestach) [inline soccer/src/soccer/planning/primitives/rrt_util.cpp]: You can delete this line.

File: soccer/src/soccer/planning/primitives/rrt_util.cpp [hunk 2]
```diff
@@ -15,40 +15,29 @@
...
-                         " grow towards given waypoints rather than towards a random point");
-    // NOLINTNEXTLINE
-    min_iterations = new ConfigInt(cfg, "PathPlanner/RRT/MinIterations", 100,
-                                   "The minimum number of iterations for running RRT");
-    // todo(Ethan) can this be increased? RRT fails sometimes. testing needed
-    // //NOLINTNEXTLINE
-    max_iterations = new ConfigInt(cfg, "PathPlanner/RRT/MaxIterations", 250,
-                                   "The maximum number of iterations for running RRT");
+
 }
```
Comment (kylestach) [inline soccer/src/soccer/planning/primitives/rrt_util.cpp]: This whole function can be removed.

**Review by kylestach — APPROVED** [review]
Just delete the test that's failing, we don't use the old config system anyway.

**Review by kasohrab — APPROVED** [review]
Looks good.

**Review by kylestach — APPROVED** [review]

**Pull request merged**